src/kuzya.cpp

```diff
@@ -1034,44 +1029,11 @@ void Kuzya::paintWarningMarkers(QList<Compiler::compilerWarning>* warningList)
 **/
 void Kuzya::slotAbout(void)
 {
-    const QString fileName = ":/AUTHORS.txt";
-    QFile file (fileName);
-    //checks file
-    if(!QFile::exists(fileName))
-    {
-        qCritical()<< "File doesn't exist"<<fileName<<endl;
-    }
-    if (!file.open(QIODevice::ReadOnly | QIODevice::Text))
-    {
-        qCritical()<<"couldn't open file"<<fileName;
-    }
-
-    QString textInf;
-    QDialog* information {new QDialog};
-    QVBoxLayout* layout {new QVBoxLayout(information)};
-    information->setLayout(layout);
-    QLabel* textInfLabel {new QLabel(information)};
-    QLabel* imgLabel {new QLabel(information)};
-
-    QTextStream in(&file);
-    textInf  = in.readAll();
-
-//set Kuzia version and build date
-    QVersionNumber verKuzia(MAJORVER, MINORVER, REVISION);
-    QDate buildDate(BUILD_YEAR, BUILD_MONTH, BUILD_DAY);
-    textInfLabel->setText(textInf.arg(verKuzia.toString(), buildDate.toString("MMMM d yyyy")));
-
-
-    imgLabel->setPixmap(QPixmap(":/common/Kuzya_about.png"));
-    layout->addWidget(imgLabel);
-    QScrollArea* scrollAreaAuthors = new QScrollArea(information);
-    scrollAreaAuthors->setWidget(textInfLabel);
-    layout->addWidget(scrollAreaAuthors);
-    information->setWindowIcon(QIcon(QDir::currentPath()+":/../../src/images/kuzya.png"));
-
-    information->exec();
-
-    delete information;
+    KuzyaAbout kuzya;
+    kuzya.checkFile();
+    kuzya.readAuthors();
```

Comment:
It is event better to pass Kuzya version,  build date and  path to AUTHORS file - as a constructor parameters. KuzyaAbout dialog will not show correct information if we not set all that, right?